apis/pkg/meta/v1alpha1/conversion.go

```diff
@@ -118,32 +113,3 @@ func (p *Provider) ConvertFrom(hub conversion.Hub) error {
 
 	return nil
 }
-
-// Hub marks this type as the conversion hub.
-func (f *Function) Hub() {}
-
-// ConvertTo converts this Configuration to the Hub version.
-func (f *Function) ConvertTo(hub conversion.Hub) error {
-	out, ok := hub.(*Function)
-	if !ok {
-		return errors.New(errWrongConvertToFunction)
-	}
-
-	conv := &GeneratedToHubConverter{}
-	*out = *conv.Function(f)
-
-	return nil
-}
-
-// ConvertFrom converts this Function from the Hub version.
-func (f *Function) ConvertFrom(hub conversion.Hub) error {
-	in, ok := hub.(*Function)
-	if !ok {
-		return errors.New(errWrongConvertFromFunction)
-	}
-
-	conv := &GeneratedFromHubConverter{}
-	*f = *conv.Function(in)
-
-	return nil
-}
```

Comment:
Don't we also want to keep meta/v1alpha1 just like provider and configuration packages? As a side note, I think we need a change in `up` cli to support the new version.

Reply:
No, I don't think so. We have `v1alpha1` for Provider and Configuration packages because there may be some packages in the wild who use the `meta.pkg.crossplane.io/v1alpha1` versions of their `crossplane.yaml` metadata. We know that's not the case with Functions.